src/Turbo/src/Twig/TurboStreamListenRendererInterface.php

```diff
@@ -23,5 +23,5 @@ interface TurboStreamListenRendererInterface
     /**
      * @param string|object $topic
      */
-    public function renderTurboStreamListen(Environment $env, $topic): string;
+    public function renderTurboStreamListen(Environment $env, $topic, array $eventSourceOptions): string;
```

Comment:
This would be a BC break.. :/

Reply:
need to add a default value, I agree. 

`public function renderTurboStreamListen(Environment $env, $topic, array $eventSourceOptions = []): string;`